### PR TITLE
URL shortener using bitly

### PR DESCRIPTION
**Pull request by gniting**
## Description
Script command that does URL shortening using Bitly.
* Will require the user to generate an access token on their bitly account. This way they can get metrics on the short URLs created. 
* No dependencies, uses curl to validate the input URL.

## Type of change
- [X] New script command

## Screenshot
image

## Dependencies / Requirements
None

## Checklist
- [X] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/browsing/shorten-url-bitly.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (unnamedd) [inline commands/browsing/shorten-url-bitly.sh:1]: Due to the need of additional modification in this Script Command, please rename it to `shorten-url-bitly.template.sh`.

**Comment by mattisssa** [comment]
Looks great! Thanks for contribution 👍

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @gniting, your Script Command looks pretty okay to me. You only need to make a small change (which is commented below) and then it will be approved! 

Thanks a lot for your contribution, one more time!

File: commands/browsing/shorten-url-bitly.sh (hunk 1 above)
Comment (gniting) [reply commands/browsing/shorten-url-bitly.sh]: Done!

**Review by unnamedd — APPROVED** [review]
Thanks @gniting!

It's approved! 
@thomaspaulmann @PitNikola @fe9lix

**Pull request merged**